build.gradle.kts

```diff
@@ -166,13 +171,20 @@ allprojects {
 				excludeGroup("ca.weblite")
 			}
 		}
+		mavenLocal {
```

Comment:
Please remove this. It's not about the scope, it's about it not being something that should be present in the `master` branch. You can keep it locally but it shouldn't be committed.